keras_nlp/src/utils/transformers/convert_gemma.py

```diff
@@ -77,10 +103,29 @@ def load_gemma_backbone(cls, preset, load_weights):
             keras_variable=decoder_layer.pre_attention_norm.variables[0],
             hf_weight_key=f"model.layers.{i}.input_layernorm.weight",
         )
-        port_weight(
-            keras_variable=decoder_layer.pre_ffw_norm.variables[0],
-            hf_weight_key=f"model.layers.{i}.post_attention_layernorm.weight",
-        )
+
+        if decoder_layer.use_post_attention_norm:
+            port_weight(
+                keras_variable=decoder_layer.post_attention_norm.variables[0],
+                hf_weight_key=f"model.layers.{i}.post_attention_layernorm.weight",
+            )
+
+        if transformers_config["model_type"] == "gemma":
+            port_weight(
+                keras_variable=decoder_layer.pre_ffw_norm.variables[0],
+                hf_weight_key=f"model.layers.{i}.post_attention_layernorm.weight",
+            )
+        elif transformers_config["model_type"] == "gemma2":
+            port_weight(
+                keras_variable=decoder_layer.pre_ffw_norm.variables[0],
+                hf_weight_key=f"model.layers.{i}.pre_feedforward_layernorm.weight",
+            )
```

Comment:
This was done in order to align the gemma 1 and gemma 2 checkpoints.

I am open to better ways to go around it.